package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.0.0",
   "private": true,
   "engines": {
-    "node": ">=18.0.0"
+    "node": ">=20.0.0"
```

Comment:
We don't need to have this key value set since it would follow whatever is set in Vercel. I left it in so enforce the node version locally.